src/main/java/duke/Task.java

```diff
@@ -0,0 +1,79 @@
+package duke;
+
+/**
+ * Represents a task in the Duke application.
+ */
+public class Task {
```

Comment:
Since Tasks are never instantiated directly and instead is solely a parent class for Deadline, Todo, and Event, you should make it an abstract class.